src/internal/umd.ts

```diff
@@ -2,13 +2,9 @@
   NOTE: This is the global export file for rxjs v6 and higher.
  */
 
-/* rxjs */
+/* rxjs and rxjs.operators */
 export * from '../index';
```

Comment:
I cannot see that an `operators` export is effected by the above export declaration, so IMO the export declaration below is needed.

Reply:
What is effected is that you will get warnings during build time.

When I updated rollup in my other PR

Reply:
Warnings or not, if you cannot point out where the `operators` export will be effected - if it's removed here - this change is unsafe.

Reply:
Had a look, the UMD output is the same. As rollup itself will not add the second export to the exports list. So they will only contain a single `exports.operators`

https://github.com/rollup/rollup/blob/273ac735b155ea63a6e8bf1d0862c872cb3e94b4/src/Graph.ts#L583-L598

Reply:
It's only the non-`compat` change that I think is unsafe. I can see where it's exported twice for the `compat` bundle and the change to the file in the `compat` directory is fine by me.

Reply:
Agreed. Updated